packages/manager/apps/pci-rancher/src/api/GET/apiv2/services.ts

```diff
@@ -41,6 +55,27 @@ export const postRancherServiceQueryKey = (rancherId: string) => [
   rancherId,
 ];
 
+export const createRancherServiceQueryKey = () => ['post/rancher/resource'];
+
+export const getRancherPlan = async () =>
+  apiClient.v2.get<RancherPlan[]>(getReferenceRancherInfo('plan'));
+
+export const getRancherVersion = async () =>
+  apiClient.v2.get<RancherVersion[]>(getReferenceRancherInfo('version'));
+
+export const createRancherService = async ({
+  projectId,
+  data,
+}: {
+  projectId: string;
+  data: CreateRancherPayload;
+}) => {
+  return apiClient.v2.post(getRancherByProjectIdQueryKey(projectId), {
+    ...data,
+    ipRestrictions: [],
+  });
+};
```

Comment:
you need to rename the api folder since you are in the GET folder or put this function in a POST folder